src/components/Snippet.vue

```diff
@@ -1,17 +1,23 @@
 <template>
-  <span
-    :class="suit()"
-    v-html="innerHTML"
+  <highlighter
+    :hit="hit"
+    :attribute="attribute"
+    :highlighted-tag-name="highlightedTagName"
+    :suit="suit"
+    highlight-property="_snippetResult"
+    pre-tag="<mark>"
```

Comment:
Thinking more, the pre tag here should be what the user or we set as highlightPreTag, no? So that would be the `__ais-highlight__`

Reply:
oh okay, I think I am missing some stuff.

![image](https://user-images.githubusercontent.com/499898/83396613-88c63a80-a3fc-11ea-814e-8e30f6a8a148.png)

I'm seeing `__ais-highlight__` in the network panel.

but when I log `hits` in the `Hits` component,

![image](https://user-images.githubusercontent.com/499898/83396762-c4f99b00-a3fc-11ea-9c51-721f090e3f0b.png)

I see `<mark>`. Do you when it's replaced by which?

Reply:
it's replaced by `escapeHighlight` in InstantSearch.js, you need to set the replaced tags there forcibly somehow to the original tags, or disable `escapeHighlight`

Reply:
@Haroenv Okay, I think I can create a pull-request in InstantSearch.js.
In the PR, there is [parseAlgoliaHit](https://github.com/algolia/vue-instantsearch/pull/783/files#diff-26a063579221d23a8e71d6870cc3c702R100) function (or whatever the name is). It behaves according to `hits.__escaped`.

If it's `true`, it will parse with

```js
export const TAG_REPLACEMENT = {
  highlightPreTag: '<mark>',
  highlightPostTag: '</mark>',
};
```

If it's `false`, it will parse with

```js
export const TAG_PLACEHOLDER = {
  highlightPreTag: '__ais-highlight__',
  highlightPostTag: '__/ais-highlight__',
};
```

Then Vue-InstantSearch uses this function.
What do you think?

Reply:
that should work, good idea. Ideally in InstantSearch.js we should change the behaviour to not just replace, but in this case it should work.

Reply:
Yes, in this way, we don't break InstantSearch.js and provide extra function for flavors. I'll continue on this path!

Reply:
https://github.com/algolia/vue-instantsearch/pull/783/commits/711134ce8e047b3ac3d0c39547a37751b2d5fece